website/docs/cloud-docs/workspaces/settings/notifications.mdx

```diff
@@ -6,6 +6,8 @@ page_title: Notifications - Workspaces - Terraform Cloud
 
 Terraform Cloud can use webhooks to notify external systems about run progress and other events. Each workspace has its own notification settings and can notify up to 20 destinations.
 
+-> Note: Notifications are not supported for workspaces configured with `Local` [execution mode](/cloud-docs/workspaces/settings#execution-mode)
```

Comment:
```suggestion
-> Note: Workspaces configured with `Local` [execution mode](/cloud-docs/workspaces/settings#execution-mode) do not support notifications.
```

Suggest reordering the sentence to avoid a passive voice style issue.